style.css

```diff
@@ -329,8 +461,82 @@ body.lock {
         display: none;
     }
     .content {
-        padding: 50px 0 0;
+        padding: 0 0;
     } 
+    .carousel__text {
+        top: 10%;
+        left: 3%;
+    }
+    .carousel__text1 {
+        color: var(--element-color);
+        font-size: 28px;
+        letter-spacing: 3px;
+    }
+    .carousel__text2 {
+        color: var(--additional-color);
+        font-size: 25px;
+        letter-spacing: 1.5px;
+    }
+    .carousel {
+        position: static; 
+        width: 100vw;
+        height: auto;
+        overflow: visible;
+    }
+    .carousel__box {
+        position: static;
+        display: inline;
+    }
+    .carousel__object {
+        width: 32%;
+        margin: 0 1px;
+    }
+    .carousel__buttons {
+        display: none;
+    }
+    .about-part {
+        display: flex; 
+        flex-direction: row;
+        height: auto;
+    }
+    .img1 {
+        left: 10%;
+    }
+    .img2 {
+        right: 10%; 
+    }
+    .about-part__img-container {
+        flex: 1 0;
+    }
+    .about-part__text-container {
+        width: 50%;
+    }
+    p {
+        text-size-adjust: auto;
+        font-family: var(--main-title-font);
+        font-size: 10px;
+        letter-spacing: 1px;
+        line-height: 2;
+    }
```

Comment:
And here you are overwriting your own code, how the turn tables have turned. This is priceless

Reply:
Removed (accidental copy-past)